src/test/java/com/google/devtools/build/lib/rules/cpp/CrosstoolConfigurationHelper.java

```diff
@@ -58,7 +58,13 @@ public static void overwriteCrosstoolWithToolchain(
 
   public static String defaultCpu() {
     if (OS.getCurrent() == OS.WINDOWS) {
-      return "x64_windows";
+      switch (CPU.getCurrent()) {
+        case X86_32:
+          return "x86_32_windows";
+        case X86_64:
+          return "x64_windows";
+        default:
+          return "unknown";
```

Comment:
This doesn't compile

Reply:
Actually, I tested it on 1.0.9 and 1.0.11, but I'll check it on freshest version.